horton/gbasis/test/test_iobas.py

```diff
@@ -139,3 +157,29 @@ def test_gobasis_contraction():
         gbc2.get_segmented_bcs()
     with assert_raises(NotImplementedError):
         gbc.normalize()
+
+
+def test_dump_basis_atom_map_gbs():
+    sto3g = GOBasisFamily('original STO-3G', filename=context.get_fn('basis/sto-3g.gbs'))
+    sto3g.load()
+    with tmpdir('horton.gbasis.test.test_iobas.test_dump_basis_atom_map_gbs') as tmp:
+        tmp_gbs = '{0}/test.gbs'.format(tmp)
+        dump_basis_atom_map_gbs(tmp_gbs, sto3g.name, sto3g.basis_atom_map)
+        new_sto3g = GOBasisFamily('new STO-3G', filename=tmp_gbs)
+        new_sto3g.load()
+        for atom, bca in new_sto3g.basis_atom_map.iteritems():
+            old_bca = sto3g.basis_atom_map[atom]
+            for i, contraction in enumerate(bca.bcs):
+                old_contraction = old_bca.bcs[i]
+                assert contraction.shell_type == old_contraction.shell_type
+                assert np.allclose(contraction.alphas, old_contraction.alphas)
```

Comment:
Why are we using allclose here? It should be the same since we're just reading in from a file? Or do we have issues with truncation? (fortran_float)